src/roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, file_path: PathType, optical_series_name: Optional[str] = "Tw
         file_path: str
             The location of the folder containing dataset.nwb file
         optical_series_name: str (optional)
```

Comment:
```suggestion
        optical_series_name: string, optional
```

This is closer to numpy docstring style right? Not that we've updated that everywhere on the repo though...

Reply:
Since the rest of this looks good, just gonna push it through